omigami/tests/spec2vec/test_spec2vec_integration.py

```diff
@@ -33,7 +33,7 @@ def test_match_spectra_from_path_negative_mode(mgf_path, spec2vec_client):
 
 @pytest.mark.internet_connection
 @pytest.mark.skip(
-    reason="this is a stress test, run only when you know what you are doing"
+    reason="this is a stress tests, run only when you know what you are doing"
```

Comment:
?

Reply:
That was Pycharms refactor tool when i moved the tests folder